applications/maplab-server-node/src/maplab-server-node.cc

```diff
@@ -1164,7 +1279,8 @@ bool MaplabServerNode::appendAvailableSubmaps() {
 void MaplabServerNode::printAndPublishServerStatus() {
   std::stringstream ss;
 
-  ss << "\033c";
+  if (FLAGS_maplab_server_clear_at_status_print)
```

Comment:
this makes it easier to debug things

Reply:
could we wrap this around with curly brackets to stay consistent?